hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh

```diff
@@ -79,13 +80,23 @@ if [ ! -d "$LOGS" ]; then
     mkdir -p "$LOGS"
 fi
 
+GREMLIN_SERVER_CONF="gremlin-server.yaml"
+if [[ $PRELOAD == "true" ]]; then
```

Comment:
how could someone load the example data directly(without docker?)

seems `-p true` is better? (and we could also support input `$PRELOAD` param)